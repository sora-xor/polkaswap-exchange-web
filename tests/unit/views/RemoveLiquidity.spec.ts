// import { shallowMount, createLocalVue } from '@vue/test-utils'
// import Vuex from 'vuex'
// import VueRouter from 'vue-router'

// import RemoveLiquidity from '@/views/RemoveLiquidity.vue'
// import { tokens } from '../../mocks/tokens'
// import { liquidities } from '../../mocks/liquidity'
// import { SoramitsuElementsImport, TranslationMock } from '../../utils'

// const localVue = createLocalVue()
// localVue.use(Vuex)
// SoramitsuElementsImport(localVue)
// localVue.use(VueRouter)
// const router = new VueRouter()

// describe('RemoveLiquidity.vue', () => {
//   let actions
//   let getters
//   let store

//   beforeEach(() => {
//     TranslationMock(RemoveLiquidity)

//     actions = {
//       getLiquidity: jest.fn(),
//       setRemovePart: jest.fn()
//     }

//     getters = {
//       liquidity: () => liquidities[0],
//       firstToken: () => tokens[0],
//       secondToken: () => tokens[1],
//       removePart: () => 100,
//       liquidityAmount: () => 100,
//       firstTokenAmount: () => 100,
//       secondTokenAmount: () => 100,
//       liquidityBalance: () => 100,
//       firstTokenBalance: () => 100,
//       secondTokenBalance: () => 100
//     }

//     store = new Vuex.Store({
//       modules: {
//         removeLiquidity: {
//           namespaced: true,
//           actions,
//           getters
//         }
//       },
//       actions: {
//         getTokens: jest.fn()
//       },
//       getters: {
//         tokens: () => tokens
//       }
//     })
//   })

//   it('should renders correctly', () => {
//     const wrapper = shallowMount(RemoveLiquidity, {
//       localVue,
//       store,
//       router
//     })
//     expect(wrapper.element).toMatchSnapshot()
//   })
// })

describe('RemoveLiquidity test', () => {
  test('Temporary test', () => {
    expect(true).toEqual(true);
  });
});
