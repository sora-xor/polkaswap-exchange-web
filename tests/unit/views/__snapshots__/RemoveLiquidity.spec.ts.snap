// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RemoveLiquidity.vue should renders correctly 1`] = `
<div
  class="remove-liquidity-container"
>
  <s-row-stub
    align="middle"
    class="header"
    flex="true"
    gutter="0"
    justify="space-between"
  >
    <s-button-stub
      borderradius="small"
      icon="arrow-left"
      nativetype="button"
      size="small"
      tabindex="0"
      tooltip=""
      tooltipplacement="top"
      type="action"
    />
     
    <div
      class="title"
    >
      
    </div>
     
    <s-tooltip-stub
      animation="el-fade-in-linear"
      borderradius="small"
      closedelay="0"
      content=""
      offset="0"
      opendelay="0"
      placement="bottom-end"
      popperclass="remove-liquidity__description"
      showarrow="true"
      tabindex="0"
      theme="light"
    >
      <s-button-stub
        borderradius="small"
        icon="info"
        nativetype="button"
        size="small"
        tabindex="0"
        tooltip=""
        tooltipplacement="top"
        type="action"
      />
    </s-tooltip-stub>
  </s-row-stub>
   
  <s-form-stub
    class="el-form--remove-liquidity"
    labelposition="right"
    validateonrulechange="true"
  >
    <!---->
     
    <div
      class="input-container"
    >
      <div
        class="input-line"
      >
        <div
          class="input-title"
        >
          
        </div>
         
        <div
          class="token-balance"
        >
          <span
            class="token-balance-title"
          >
            
          </span>
           
          <span
            class="token-balance-value"
          >
            1000.00
          </span>
        </div>
      </div>
       
      <div
        class="input-line"
      >
        <s-form-item-stub
          prop=""
          showmessage="true"
        >
          <s-input-stub
            accept=""
            autocomplete="off"
            borderradius="small"
            disabled="true"
            form=""
            label=""
            name=""
            placeholder="0.00"
            prefix=""
            size="big"
            suffix=""
            tabindex=""
            type="text"
            value="0.00"
          />
        </s-form-item-stub>
         
        <div
          class="token"
        >
          <s-button-stub
            borderradius="small"
            class="el-button--max"
            icon=""
            nativetype="button"
            size="small"
            tabindex="0"
            tooltip=""
            tooltipplacement="top"
            type="tertiary"
          >
            
            
          
          </s-button-stub>
           
          <s-button-stub
            borderradius="small"
            class="el-button--choose-token"
            icon=""
            nativetype="button"
            size="small"
            tabindex="0"
            tooltip=""
            tooltipplacement="top"
            type="tertiary"
          >
            <div
              class="liquidity-logo"
            >
              <!---->
            </div>
            
            XOR-KSM
          
          </s-button-stub>
        </div>
      </div>
    </div>
     
    <s-icon-stub
      class="icon-divider"
      name="arrow-bottom"
      size="medium"
    />
     
    <div
      class="input-container"
    >
      <div
        class="input-line"
      >
        <div
          class="input-title"
        >
          
        </div>
         
        <div
          class="token-balance"
        >
          <span
            class="token-balance-title"
          >
            
          </span>
           
          <span
            class="token-balance-value"
          >
            10000.00
          </span>
        </div>
      </div>
       
      <div
        class="input-line"
      >
        <s-form-item-stub
          prop=""
          showmessage="true"
        >
          <s-input-stub
            accept=""
            autocomplete="off"
            borderradius="small"
            disabled="true"
            form=""
            label=""
            name=""
            placeholder="0.00"
            prefix=""
            size="big"
            suffix=""
            tabindex=""
            type="text"
            value="0.00"
          />
        </s-form-item-stub>
         
        <div
          class="token"
        >
          <s-button-stub
            borderradius="small"
            class="el-button--max"
            icon=""
            nativetype="button"
            size="small"
            tabindex="0"
            tooltip=""
            tooltipplacement="top"
            type="tertiary"
          >
            
            
          
          </s-button-stub>
           
          <s-button-stub
            borderradius="small"
            class="el-button--choose-token"
            icon=""
            nativetype="button"
            size="small"
            tabindex="0"
            tooltip=""
            tooltipplacement="top"
            type="tertiary"
          >
            <!---->
            
            XOR
          
          </s-button-stub>
        </div>
      </div>
    </div>
     
    <s-icon-stub
      class="icon-divider"
      name="plus"
      size="medium"
    />
     
    <div
      class="input-container"
    >
      <div
        class="input-line"
      >
        <div
          class="input-title"
        >
          <span>
            
          </span>
        </div>
         
        <div
          class="token-balance"
        >
          <span
            class="token-balance-title"
          >
            
          </span>
           
          <span
            class="token-balance-value"
          >
            1000.00
          </span>
        </div>
      </div>
       
      <div
        class="input-line"
      >
        <s-form-item-stub
          prop=""
          showmessage="true"
        >
          <s-input-stub
            accept=""
            autocomplete="off"
            borderradius="small"
            disabled="true"
            form=""
            label=""
            name=""
            placeholder="0.00"
            prefix=""
            size="big"
            suffix=""
            tabindex=""
            type="text"
            value="0.00"
          />
        </s-form-item-stub>
         
        <div
          class="token"
        >
          <s-button-stub
            borderradius="mini"
            class="el-button--max"
            icon=""
            nativetype="button"
            size="small"
            tabindex="0"
            tooltip=""
            tooltipplacement="top"
            type="tertiary"
          >
            
            
          
          </s-button-stub>
           
          <s-button-stub
            borderradius="small"
            class="el-button--choose-token"
            icon=""
            nativetype="button"
            size="small"
            tabindex="0"
            tooltip=""
            tooltipplacement="top"
            type="tertiary"
          >
            <!---->
            
            KSM
          
          </s-button-stub>
        </div>
      </div>
    </div>
     
    <s-button-stub
      borderradius="medium"
      disabled="true"
      icon=""
      nativetype="button"
      size="medium"
      tabindex="0"
      tooltip=""
      tooltipplacement="top"
      type="primary"
    >
      
        
      
    </s-button-stub>
  </s-form-stub>
   
  <s-row-stub
    align="top"
    class="price-container"
    flex="true"
    gutter="0"
    justify="space-between"
  >
    <div>
      
    </div>
     
    <div>
      <div>
        1 XOR = 10018.18 KSM
      </div>
       
      <div>
        1 KSM = 0.00 XOR
      </div>
    </div>
  </s-row-stub>
</div>
`;
